src/plugins/kibana/public/visualize/styles/main.less

```diff
@@ -2,21 +2,247 @@
 @import (reference) "~bootstrap/less/list-group";
 @import (reference) "~ui/styles/list-group-menu";
 
+@icon-background-color: #416376;
```

Comment:
Whats the deal with this color? Do we use it anywhere else? If so, let's use a variable, if not, let's find a color we do use elsewhere and use its variable